### ISSUE TITLE
Advanced automations around current app and play/pause state

### ISSUE THREAD

**Issue by beslergr** [maxileith/homebridge-appletv-enhanced#221]
### Current Situation

Would like the ability to add advanced automations surrounding the currently open app, and the play/pause state.

### Proposed Change

For instance, if I’m in the Netflix app, the automatically playing trailer at the top of the screen can cause issues because the plugin reads this as the “playing” state, not the “pause” state. Would like to use a personal Shortcut to make a delay for this state, specific to the app. But that’s not an issue, obviously, with say, YouTube TV.

Would also request the Home Screen or Screen Saver be included in this as well.

Basically trying to turn on the lights in the “pause” state, but not wanting that to occur in certain apps.

**Comment by maxileith** [comment]
> Would like to use a personal Shortcut to make a delay for this state, specific to the app. 

This is already possible by checking the active identifier attribute of the Apple TV accessory in your shortcut.

**Comment by beslergr** [comment]
Actually, was able to figure this out. Only want to use it on a few apps, as some of these have those previews right when you load the app that cause the motion sensor to switch to Playing. Would love if apps used the "Idle" command ...

**Comment by maxileith** [comment]
> Would love if apps used the "Idle" command ...

The plugin just reports the state that the Apple TV itself is reporting, so I do not have any influence on that.

**Comment by beslergr** [comment]
Yeah, figured it’s on the app side of things. Hopefully devs will start utilizing it!

**Comment by DataFran** [comment]
Hi, I'm trying to write an automation to exclude my "play, pause, stop" scenes from certain apps (namely Fitness+), and found this issue which seems to address a similar problem. When I go into Shortcuts and pull Active Identifier, it's a list of numbers. How do I find the active identifier I'm looking for?
